dev/ci/ci-menhir.sh

```diff
@@ -10,8 +10,10 @@ git_download menhirlib
 if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
 ( cd "${CI_BUILD_DIR}/menhirlib"
-  sed -i.bak "s/unreleased/$(date +%Y%m%d)/" dune-project
-  echo "Definition require_$(date +%Y%m%d) := tt." > coq-menhirlib/src/Version.v
+  if grep -q unreleased dune-project; then
+    sed -i.bak "s/unreleased/$(date +%Y%m%d)/" dune-project
+    echo "Definition require_$(date +%Y%m%d) := tt." > coq-menhirlib/src/Version.v
```

Comment:
Seems very unlikely the day changes between the two commands, but just to be safe, why not call `date` only once and put the result in some variable?

Reply:
changed